### DISCUSSION TITLE
Wind gusts for current conditions not reported

### DISCUSSION THREAD

**Discussion by sdennstaedt** [weather-gov/api#390] [Q&A]
The wind **gusts** in this report are not being carried into the json below.  Gusts are showing as null although the gusts are showing as 20 knots in the report.

https://api.weather.gov/stations/kmsy/observations/latest?require_qc=true
    
"properties": {
        "@id": "https://api.weather.gov/stations/KMSY/observations/2019-10-21T17:53:00+00:00",
        "@type": "wx:ObservationStation",
        "elevation": {
            "value": 9,
            "unitCode": "unit:m"
        },
        "station": "https://api.weather.gov/stations/KMSY",
        "timestamp": "2019-10-21T17:53:00+00:00",
        "rawMessage": "KMSY 211753Z 22012G20KT 2SM TSRA BR SCT015CB SCT026 BKN037 OVC130 24/22 A2988 RMK AO2 LTG DSNT ALQDS SLP120 OCNL LTGICCG OHD TS OHD MOV NE P0017 60021 T02440222 10261 20244 51012",
        "textDescription": "Thunderstorms and Rain and Fog/Mist",
        "icon": "https://api.weather.gov/icons/land/day/rain?size=medium",
        "presentWeather": [
            {
                "intensity": null,
                "modifier": null,
                "weather": "thunderstorms",
                "rawString": "TS"
            },
            {
                "intensity": null,
                "modifier": null,
                "weather": "rain",
                "rawString": "TSRA"
            },
            {
                "intensity": null,
                "modifier": null,
                "weather": "fog_mist",
                "rawString": "BR"
            }
        ],
        "temperature": {
            "value": 24.400000000000034,
            "unitCode": "unit:degC",
            "qualityControl": "qc:V"
        },
        "dewpoint": {
            "value": 22.200000000000045,
            "unitCode": "unit:degC",
            "qualityControl": "qc:V"
        },
        "windDirection": {
            "value": 220,
            "unitCode": "unit:degree_(angle)",
            "qualityControl": "qc:V"
        },
        "windSpeed": {
            "value": 6.2000000000000002,
            "unitCode": "unit:m_s-1",
            "qualityControl": "qc:V"
        },
        "windGust": {
            "value": null,
            "unitCode": "unit:m_s-1",
            "qualityControl": "qc:Z"
        },
        "barometricPressure": {
            "value": 101190,
            "unitCode": "unit:Pa",
            "qualityControl": "qc:V"
        },

**Comment by jah1007** [comment]
This typically means that QC failed at the upstream provider of this data, MADIS.

**Comment by StephenClouse** [comment]
We don't actually parse the METAR data in API. We rely on [MADIS](https://madis.ncep.noaa.gov/) to feed us all the observation data (both METAR and non-METAR) in a consistent format.

We're forwarding this to the MADIS team to determine why it does not appear to have parsed properly.

**Comment by kollymars** [comment]
MADIS application released a new version of their software that fixed a bug on their end.  gust is now reported on API.  https://api.weather.gov/stations/KMSY/observations/2025-10-30T15:50:00+00:00
